app/src/main/java/blockeq/com/stellarwallet/services/networking/Horizon.kt

```diff
@@ -144,6 +145,51 @@ class Horizon {
             }
         }
 
+        class ChangeTrust(private val listener: SuccessErrorCallback, private val asset: Asset,
+                          private val removeTrust: Boolean)
+            : AsyncTask<Void, Void, ErrorResponse>() {
+
+            override fun doInBackground(vararg params: Void?): ErrorResponse? {
+                Network.usePublicNetwork()
+
+                val server = Server(PROD_SERVER)
+                val sourceKeyPair = WalletApplication.session!!.keyPair
+                val limit = if (removeTrust) {
```

Comment:
An alternative suggestion to keep it concise: `val limit = removeTrust ? "0.0000000" : Constants.MAX_ASSET_STRING_VALUE`

Reply:
Yup I'll do that